backend/src/main/java/org/example/backend/MovieController.java

```diff
@@ -30,4 +32,13 @@ public Movie getMovieById(@PathVariable String id) {
     public void deleteMovieById(@PathVariable String id) {
         movieService.deleteMovie(id);
     }
-}
\ No newline at end of file
+
+    @PutMapping(path = {"{id}"})
```

Comment:
```suggestion
    @PutMapping({"/{id}"})
```
path angabe nicht nötig